src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1550,6 +1550,12 @@ public Article updateArticle(Article article) {
                 JSON, json(Collections.singletonMap("article", article))), handle(Article.class, "article")));
     }
 
+    public Translation createArticleTranslation(Long articleId, Translation translation) {
+        checkHasArticleId(articleId);
```

Comment:
Why do we validate the Id of the article while in updateArticleTranslation we are verifying the Id of translation ?